micro-rdk/src/esp32/tls.rs

```diff
@@ -91,22 +93,18 @@ pub struct Esp32TLSStream {
     socket: Async<TcpStream>, // may store the raw socket
 }
 
+#[derive(Clone)]
 pub struct Esp32TLSServerConfig {
-    srv_cert: Vec<u8>,
-    srv_key: *const u8,
-    srv_key_len: u32,
+    pub srv_cert: Vec<u8>,
```

Comment:
This is now identical to `NativeTLSServerConfig` so it should probably be commoned out.